implementations/rust/ockam/ockam_api/src/nodes/models/services.rs

```diff
@@ -356,7 +358,7 @@ impl<'a> StartOktaIdentityProviderRequest<'a> {
     }
 }
 
-#[derive(Debug, Clone, Decode, Encode)]
+#[derive(Debug, Clone, Serialize, Decode, Encode)]
```

Comment:
The `tag` field must have the `#[serde(skip_serializing)]` attribute so it can compile when the `tag` feature is enabled.

Reply:
Added the attributes in the latest commit, but I'm not sure if they're required for the `new` methods implemented for the structs or just for the structs themselves. 

Let me know if this solves it (I added it to both in this case)

Reply:
I forgot to mention in my previous command how to manually check the `tag` feature: `cargo build --features tag`. There was a small issue with the attributes, but since it was a small change, I fixed it myself.